Composer/plugins/localPublish/src/index.ts

```diff
@@ -210,7 +211,7 @@ class LocalPublisher {
 
       // take the 0th item off the array, leaving just the args
       const startCommand = commandAndArgs.shift();
-
+      
```

Comment:
kill this?

Reply:
Ha, I did this on purpose because I love spaces in code. It's the C++ code in me. I'll let the js flow through my veins, removing the space now :)